llm_bench/python/benchmark.py

```diff
@@ -40,6 +41,10 @@
 stable_diffusion_hook = StableDiffusionHook()
 
 
+def get_git_revision_hash() -> str:
+    return subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode('ascii').strip()
```

Comment:
When git is not found, or git rev-parse HEAD returns nothing or error, we should do nothing here.
Can we use git hooks to save a version.txt under top folder, and print version.txt here if it exists?